dropgz/build/linux.Dockerfile

```diff
@@ -12,7 +12,7 @@ COPY ./azure-ipam .
 RUN curl -LO --cacert /etc/ssl/certs/ca-certificates.crt https://github.com/Azure/azure-container-networking/releases/download/azure-ipam%2F$AZIPAM_VERSION/azure-ipam-$OS-$ARCH-$AZIPAM_VERSION.tgz && tar -xvf azure-ipam-$OS-$ARCH-$AZIPAM_VERSION.tgz
 
 FROM tar AS azure-vnet
-ARG AZCNI_VERSION=v1.5.4
+ARG AZCNI_VERSION=v1.5.8
```

Comment:
We recently added the dropgz for windows too. So would be better to update the version for that as well: https://github.com/Azure/azure-container-networking/blob/master/dropgz/build/windows.Dockerfile#L9